.travis.yml

```diff
@@ -0,0 +1,7 @@
+language: shell
```

Comment:
We need to install `jq`, I doubt it's available by default

Reply:
Actually Travis has it pre-installed.